### PR TITLE
Recipe Ingredients - UI changes

### PR DESCRIPTION
**Pull request by LeCitrus**
originally tests changed to UI changes

### CODE REVIEW

File: app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java [hunk 1]
```diff
@@ -25,12 +31,21 @@ public class RecipeAddEditIngredientActivity extends AppCompatActivity {
...
-    EditText amount;
-    EditText unit;
-    EditText category;
+    EditText description, amount, unit, category;
     TextView activity_title;
 
+    // https://developer.android.com/reference/android/widget/AutoCompleteTextView
+    // Accessed October 30, 2022
+    // https://en.wikibooks.org/wiki/Cookbook:Units_of_measurement
+    private static final String[] unitAC = new String[] {
```
Comment (joshuanianji) [inline app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java:40]: LET'S GOOOOOO

File: app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java [hunk 2]
```diff
@@ -44,34 +59,90 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
-        Button cancelButton = findViewById(R.id.cancel_button);
-        cancelButton.setOnClickListener(v -> {
-            Log.i(TAG + ":cancel", "Cancel ingredient add");
-            setResult(Activity.RESULT_CANCELED);
-            finish();
-        });
+        if (invalidInput){
+            // If blanks, only print blank messages
+            Toast toast = Toast.makeText(getApplicationContext(), String.join(", ", toastMessage) + " must be filled!", Toast.LENGTH_LONG);
```
Comment (joshuanianji) [inline app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java:131]: Can you change this toast to a snackbar? Snackbars should be a bit more consistent across the system, and has a better UX.

https://stackoverflow.com/a/34432504

**Review by joshuanianji — CHANGES_REQUESTED** [review]
Good stuff 😎, just a simple change!

File: app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java (hunk 2 above)
Comment (LeCitrus) [reply app/src/main/java/com/androidimpact/app/activities/RecipeAddEditIngredientActivity.java:131]: Hey Josh, thanks for sending this! I agree snackbars are better as they get dismissed when switching activities.

**Review by joshuanianji — APPROVED** [review]
Nice, LGTM

**Pull request merged**